db/migrate/20200415150049_set_provider_name_search_on_provider.rb

```diff
@@ -0,0 +1,12 @@
+class SetProviderNameSearchOnProvider < ActiveRecord::Migration[6.0]
+  def up
```

Comment:
I can see the case for using a migration to add / update columns (as above) but not sure it should be used to populate data - I would use a rake task for this